### PR TITLE
Fix UUID generation

### LINKED ISSUE #21: Always sets a new Randomly generated UUID
When using [deploy](https://github.com/PowerLoom/deploy) `build-dev.sh` setup, local consensus service is not picking up the provided UUID, it instead generates and add a randomly generated UUID for snapshotter.

### PR DESCRIPTION
**Pull request by SwaroopH**
Now uses environment file from this repo instead of deploy dev to keep things consistent.

Also fixes #21

### CODE REVIEW

File: init_processes.sh [hunk 1]
```diff
@@ -1,20 +1,31 @@
 #!/bin/bash
 
+if [ -f ".env" ]; then
+    source .env
```
Comment (xadahiya) [inline init_processes.sh:4]: `source` is not available in docker container for some reason, throwing `source not found` error. Pushing a fix then approving.

Comment (xadahiya) [reply init_processes.sh]: Fixed, working fine now.

**Comment by xadahiya** [comment]
Not sure what the reason is for this but logs inside the docker container are a little messed up. See the process names @SwaroopH 
Screenshot 2023-03-07 at 3 54 31 PM
Screenshot 2023-03-07 at 3 54 40 PM

**Review by xadahiya — APPROVED** [review]
Fixed some minor issues in last 2 commits. Tested everything locally, looks good now. @SwaroopH do take a look once before merging though.

**Comment by SwaroopH** [comment]
> Not sure what the reason is for this but logs inside the docker container are a little messed up. See the process names @SwaroopH Screenshot 2023-03-07 at 3 54 31 PM Screenshot 2023-03-07 at 3 54 40 PM

Ah right, I should have added a prefix/namespace to the env variables. PM2 is picking up the `name` and overriding what is defined in our config, very strange prioritization!

**Pull request merged**